index.Rmd

```diff
@@ -106,24 +106,24 @@ After completing the course, you will be able to do the following:
 
 ## Theory Construction and Statistical Modeling
 
-In order to test a theory, we must express the theory as a statistical model, and then test this model on quantitative (numeric) data. This course uses existing tutorial datasets to practice the process of translating verbal theories into testable statistical models. Those who are interested in the methods of acquiring high quality data for your own theory, we refer to the course “Conducting a Survey” which is taught from November – January.
+In order to test a theory, we must express the theory as a statistical model and then test this model on quantitative (numeric) data. This course uses existing tutorial datasets to practice the process of translating verbal theories into testable statistical models. Those who are interested in the methods of acquiring high quality data for your own theory, we refer to the course “Conducting a Survey” which is taught from November – January.
 In this course we will use datasets from different disciplines within the social sciences (educational sciences, psychology and sociology) to explain and illustrate theories and practices that are used in all social science disciplines to statistically model social science theories.
 Most information about the course is available in this GitBook.
 Communication about the course occurs through <https://uu.blackboard.nl> (Login with your student ID and password).
 
 ## Course overview
 
-The course consists of three parts, each of which are examinated with an assignment:
+The course consists of three parts, each of which is assessed with an assignment:
 
 1. Factor analysis, in which you learn different ways of capturing unobserved constructs.
 2. Path analysis, in which you learn how to model regression and ANOVA as a structural equation model with observed variables.
 3. Structural equation modeling, in which you put together the first two topics.
 
 ## Adaptations related to the coronavirus measures
 
-The corona crisis is challenging all of us to rethink how we teach and learn. But aside from the challenges, it also offers opportunities. In adapting this course, we kept two goals in mind: To increase the alignment between the way of teaching and the learning goals, and second, to ensure high-quality interaction among students and between the students and teachers while still using online communication. Based on these goals, we made the following changes:
+The corona crisis is challenging all of us to rethink how we teach and learn. But aside from the challenges, it also offers opportunities. In adapting this course, we kept two goals in mind: increasing the alignment between the way of teaching and the learning goals, and insuring high-quality interaction among students and between the students and teachers while still using online communication. Based on these goals, we made the following changes:
```

Comment:
The correct term is ensure, I'm certain

Reply:
I was just trying to make the sentence shorter, but I grammar-ed it up all wrong in the process!  🤣